util/fpga/splice_nexysvideo.sh

```diff
@@ -14,9 +14,10 @@
 #  lowrisc_systems_top_earlgrey_nexysvideo_0.1.splice.bit
 set -e
 
-BUILD_DIR=build-fpga
-TARGET_PREFIX="$BUILD_DIR/sw/device/boot_rom/boot_rom"
-#TARGET_PREFIX="sw/${BUILD_DIR}/rom"
+. ci/build_consts.sh
```

Comment:
nit again on the `ci` file outside of CI, might `util` be a better place for `build_consts.sh`?